www/src/views/builder/index.ejs

```diff
@@ -321,23 +321,36 @@
 											<option value="3">Shield slot only</option>
 										</select>
 									</div>
+                                </div>
+								<div class="col-auto" ng-if="currentItem.slot == 13 || currentItem.slot == 2">
+									<div class="input-group input-group-sm">
+										<div class="input-group-prepend">
+										</div>
+										<!-- Button trigger modal -->
+										<button type="button" class="btn btn-primary" data-toggle="modal" ng-if="!isRuneCrafting" ng-click="runeCraftButtonClick()" data-target="#runeCraftCharmModal">
+											Runecraft Customizer
+										</button>
+										<button type="button" class="btn btn-primary" data-toggle="modal" ng-if="isRuneCrafting" ng-click="runeCraftButtonClick()" data-target="#runeCraftCharmModal">
+											Items
+										</button>
+									</div>
                                 </div>
                             </div>
 							<div class="row">
 								<div class="table-responsive" ng-cloak>
 									<table class="table table-striped table-bordered table-sm table-hover">
```

Comment:
I think it might be best to just have two tables here, one for items and one for runecrafting, and put the `ng-if` on both tables based on which one should render. That way the logic for each of these tables stays as simple as possbile.